servers/services/src/main/java/org/projectnessie/services/impl/NamespaceApiImpl.java

```diff
@@ -262,11 +276,11 @@ public void updateProperties(NamespaceParams params, NamespaceUpdate namespaceUp
 
   private Stream<KeyEntry<Content.Type>> getNamespacesKeyStream(
       @Nullable Namespace namespace,
-      BranchName branch,
+      WithHash<NamedRef> refWithHash,
```

Comment:
The function only uses the `Hash` - maybe better to refactor to `Hash hash` (the name's irrelevant)